### ISSUE TITLE
video: exit full screen mode, lost focus

### ISSUE THREAD

**Issue by shanelin1** [eBay/ebayui-core#1442]
# Bug Report

## Description
in iOS + safari with <strike>talkback</strike> voiceover enabled, play video in fullscreen, swipe to close button in fullscreen, double tap to close/exit fullscreen mode, the url bar has the focus.  Need some kind of "exit" event to notify the parent page so that they can handle the focus on correct element,

## Screenshots
![image](https://user-images.githubusercontent.com/87088707/124824805-b77be200-df27-11eb-956e-f37f8920898d.png)

**Comment by agliga** [comment]
Will investigate.
However IOS does not have fullscreen controls as well as other platforms. We might have to rely on the native experience

**Comment by agliga** [comment]
Closing, this is a shaka player issue
